infra/app/function.bicep

```diff
@@ -113,15 +113,25 @@ module searchRoleFunction '../core/security/role.bicep' = if (authType == 'rbac'
 }
 
 // Storage Blob Data Contributor
-module storageRoleFunction '../core/security/role.bicep' = if (authType == 'rbac') {
-  name: 'storage-role-function'
+module storageBlobRoleFunction '../core/security/role.bicep' = if (authType == 'rbac') {
+  name: 'storage-blob-role-function'
   params: {
     principalId: function.outputs.identityPrincipalId
     roleDefinitionId: 'ba92f5b4-2d11-453d-a403-e96b0029c9fe'
     principalType: 'ServicePrincipal'
   }
 }
 
+// Storage Queue Data Contributor
+module storageQueueRoleFunction '../core/security/role.bicep' = {
```

Comment:
I think this is a bug - I think this should have an `if (authType == 'rbac')` clause

Reply:
Fixed now